### ISSUE TITLE
Add Dependency Dashboard showcase page

### ISSUE THREAD

**Issue by HonkingGoose** [renovatebot/renovate#11178]
**What would you like Renovate to be able to do?**

A user [commented](
https://github.com/renovatebot/renovate/issues/11134#issuecomment-894242852) about our Dependency Dashboard:

>Awesome feature. It should have a better exposure in the documentation maybe with a dedicated section in getting started.

I also think the dashboard is a great feature, and I agree that we should promote it better, and increase the discoverability of the feature.

We need this page to be there, before we can make "Dependency Dashboard enabled" the default behavior in `v26`.

**Did you already have any implementation ideas?**

It would probably go well under the Getting Started section of the sidebar, with a new entry "Dependency Dashboard".

I could write sections on:

- Supported platforms
- How to enable the dashboard
- Use-cases:
  - Dependency Dashboard Approval workflow
  - Visibility into rejected/deferred updates
  - Other things that I'm not thinking of right now? Open for suggestions! 😉 
  
Let me know if this is something you want me to work on. And if you have ideas on how to make it better, please share

**Comment by HonkingGoose** [comment]
@rarkins Do you want me to target the `main` or `v26` branch with a PR for this page?

**Comment by rarkins** [comment]
I think main is ok, thanks - as long as it doesn't say (yet) that it's enabled by default

**Comment by rowanbeentje** [comment]
Could there be anything around recommended usage? For example, one slight gotcha with the dependency dashboard for me is that there's no kind of notification possible when the dashboard is updated (see https://github.com/isaacs/github/issues/310) - so I believe that with it enabled, the visibility of new PRs moves to an issue which developers need to manually check frequently to see if there are any warnings, errors, or dependencies that need updating.

**Comment by HonkingGoose** [comment]
> Could there be anything around recommended usage? For example, one slight gotcha with the dependency dashboard for me is that there's no kind of notification possible when the dashboard is updated (see [isaacs/github#310](https://github.com/isaacs/github/issues/310)) - so I believe that with it enabled, the visibility of new PRs moves to an issue which developers need to manually check frequently to see if there are any warnings, errors, or dependencies that need updating.

That is the tradeoff when you enable "Dependency Dashboard approval" for specific updates/types of updates/all updates. For those updates you tell Renovate to shut up and wait for approval, you need to check yourself if there are any updates.

I don't think many people would benefit from telling Renovate to "go sit in a corner and wait quietly" though. The whole point of having a bot is that you're notified of updates when they happen... 😄

**Comment by rarkins** [comment]
Yes, I think we should cover things, namely the threat of "out of sight, out of mind":
- Dependency Dashboard Approval is far superior to disabling (e.g. for major updates) because at least you can fully see what's pending instead of it being invisible
- Putting all on Approval is not recommended because many or most repo users won't ever look there regularly enough

**Comment by HonkingGoose** [comment]
> Yes, I think we should cover things, namely the threat of "out of sight, out of mind":
> 
>     * Dependency Dashboard Approval is far superior to disabling (e.g. for major updates) because at least you can fully see what's pending instead of it being invisible
> 
>     * Putting all on Approval is not recommended because many or most repo users won't ever look there regularly enough

I've added your comments to the draft PR at #11183.
